chat/toxcore/patches/patch-testing_misc__tools.c

```diff
@@ -0,0 +1,13 @@
+$NetBSD$
+
+--- testing/misc_tools.c.orig	2018-08-16 20:45:33.000000000 +0000
++++ testing/misc_tools.c
+@@ -27,7 +27,7 @@
+ 
+ #ifndef _POSIX_C_SOURCE
+ // For nanosleep().
+-#define _POSIX_C_SOURCE 199309L
++#define _POSIX_C_SOURCE 200112L
```

Comment:
Why this?
If we want to overwrite or define `_POSIX_C_SOURCE` we shall do it from main package's level with `-D_POSIX_C_SOURCE=200112L`

Reply:
Done.